File: scala-unit-tests/src/main/scala/first_module/BasicCalcFunc.scala
package first_module

// Самые простые функции для демонстрации базового функционала Unit тестов
def add(x: Int, y: Int): Int = x + y

def sub(x: Int, y: Int): Int = x - y

def mul_by_2(x: Int): Int =
  // Случай, который должен быть выявлен в результате прохождения тестов
  x * 3

File: python-unit-tests/my_package/first_module/basic_calc_func.py
# Самые простые функции для демонстрации базового функционала Unit тестов
def add(x: int, y: int) -> int:
    return x + y


def sub(x: int, y: int) -> int:
    return x - y


def mul_by_2(x: int) -> int:
    # Случай, который должен быть выявлен в результате прохождения тестов
    return x * 3
